setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = aiohttp-cors
+version = 0.7.0
+author = Vladimir Rutsky and aio-libs team
+author_email = vladimir@rutsky.org
+license = Apache-2.0
+description = CORS support for aiohttp
+long_description = file: README.rst
```

Comment:
Now `setup.py` merges `README` and `CHANGES` in long description

Reply:
Yes it does, bur setuptools cannot do that. I guess moving the metadata into a declarative config worth to keeping only package description.

Reply:
If declarative config doesn't support the feature -- please use existing approach